.env.example

```
@@ -1,2 +1,12 @@
 POLAR_API_BASE_URL=https://api.polar.sh/v1
-LANGBASE_API_KEY="your_langbase_api_key_here"
\ No newline at end of file
+LANGBASE_API_KEY="your_langbase_api_key_here"
+
+DATABASE_URL="your_database_connection_string_here"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)